al_aws_collector.js

```diff
@@ -107,6 +107,7 @@ class AlAwsCollector {
             formatFun, healthCheckFuns, statsFuns) {
         this._invokeContext = context;
         this._arn = context.invokedFunctionArn;
+        this._awsAccountId = m_alAws.arnToAccId(this._arn);
```

Comment:
please use `context.invokedFunctionArn` here.
This is an anti pattern to use refer to `this` in constructor.